website/docs/reference/technical-debt.md

```diff
@@ -3,48 +3,44 @@ title: Technical Debt
+Marking a flag as stale generates a `feature-stale-on` [event](/reference/events#feature-stale-on). You can use [an integration](/reference/integrations) to trigger automated workflows, such as posting notifications in a Slack channel, breaking project builds if the code contains stale flags, or automatically opening pull requests to remove stale flags from the code.
 
-- **Release** - Used to enable trunk-based development for teams practicing Continuous Delivery. _Expected lifetime 40 days_
-- **Experiment** - Used to perform multivariate or A/B testing. _Expected lifetime 40 days_
-- **Operational** - Used to control operational aspects of the system's behavior. _Expected lifetime 7 days_
-- **Kill switch** - Used to to gracefully degrade system functionality. _(permanent)_
-- **Permission** - Used to change the features or product experience that certain users receive. _(permanent)_
+To find stale and potentially stale flags in a project, apply the **State** filter in the **Feature flags** list.
 
-If your flag exceeds the expected lifetime of its flag type it will be marked as _potentially stale_.
+While a flag's state does not affect its behavior in applications, using states to manage flags helps reduce technical debt and maintain a cleaner codebase.
 
-One thing to note is that the report card and corresponding list are showing stats related to the currently selected project. If you have more than one project, you will be provided with a project selector in order to swap between the projects.
+## Project status
 
-### Health rating
+Each project has a **Project status** dashboard, where you can view its health status and the total number of unhealthy flags. All active flags are considered healthy, while stale and potentially stale flags are considered unhealthy. To keep your project in a healthy state, [archive stale feature flags](/reference/feature-toggles#archive-a-feature-flag) and remove code from your codebase.
 
-Unleash calculates a project's health rating based on the project's total number of active flags and how many of those active flags are stale or potentially stale. When you archive a flag, it no longer counts towards your project's health rating.
+![Project status dashboard](/img/project-status-dashboard.png)
 
-The health rating updates once every hour, so there may be some lag if you have recently added, removed, or changed the status of a flag.
+Your overall project health rating is the percentage of healthy flags in your project. To view your project health over time, go to [Insights](/reference/insights).
 
-### Flag list {#toggle-list}
+:::info
 
-![A table of the flags in the current project with their health reports. The table has the following columns: name, last seen, created, expired, status, and report.](/img/togglelist.png)
+Project status updates every hour. Changes to flags, such as adding, removing, or changing their state, may take some time to reflect.
+
+:::
```

Comment:
Ah, this isn't correct. The project status tab calculates everything on the fly, so it should change immediately when you update your resources.

Reply:
Ok, thanks!